src/network.rs

```diff
@@ -142,7 +142,7 @@ pub(crate) unsafe extern "C" fn network_open(
 }
 
 /// Where the ACTUAL work happens, taking advantage of Rust error-handling etc
-fn _network_open(
```

Comment:
See my comment below regarding the changes in this file: https://github.com/georust/proj/pull/72/files#r574620593